### ISSUE TITLE
Brave Keystrokes not captured

### ISSUE THREAD

**Issue by Caliv0** [monkeytypegame/monkeytype#5179]
### Did you clear cache before opening an issue?

- [X] I have cleared my cache

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does the issue happen when logged in?

Yes

### Does the issue happen when logged out?

Yes

### Does the issue happen in incognito mode when logged in?

Yes

### Does the issue happen in incognito mode when logged out?

Yes

### Current Behavior

When I start to type, nothing happens. Even f5 doesn't work. The page is not frozen, but rather every input is captured and then ignored.

### Expected Behavior

It should not ignore my inputs

### Steps To Reproduce

I'm using Brave. On Ungoogled-Chromium the issue doesn't occur.

1. Open monkeytype.com
2. Click in the center of the page
3. Start typing

If you opened the page for the first time, the bug probably occurs.
After a reload, the bug is gone, probably due to the faster loading speed because of browser caching.
To have the bug consistently, one can disable cache in the dev-tools. (tab) Network -> (checkbox) Disable cache.

The issue happened very consistently when I had the language tool spellchecking extension activated as well.

### Environment

- OS: Arch Linux
- Browser: Brave
- Browser Version:  122.1.63.165

### Anything else?

This is what the network tab looks like when the bug occurs.
![image](https://github.com/monkeytypegame/monkeytype/assets/55995344/9a1322d2-aac9-4ac3-8c5e-efaf69829e23)

The issue does occur even without any extensions installed.

**Comment by Miodec** [comment]
What do you get when you shift click the version number. This issue was fixed yesterday.

**Comment by Caliv0** [comment]
While the bug is active, nothing. It is just ignored.

After some attempts with cache enabled, I get this
![image](https://github.com/monkeytypegame/monkeytype/assets/55995344/b83cb0c9-5921-47c2-b3ed-9a8f0e729492)

**Comment by Miodec** [comment]
Im on the same version, everything is fine. Disable your extensions and try again.

**Comment by Caliv0** [comment]
When I'm in incognito mode everything wooks well. I think the theme is partially responsible for the issue. I was unable to reproduce this in incognito (with all extensions disabled) but once I set the theme again to "bushido" the issue occured again. You might have to open and close the tab a few times for the issue to occur again.

**Comment by Miodec** [comment]
What are you brave shield settings? Themes can't be the issue here.

**Comment by Caliv0** [comment]
![image](https://github.com/monkeytypegame/monkeytype/assets/55995344/e7f91f70-648b-42ec-b38c-9e729b5548ac)

Since disabling shield, I was unable to reproduce the issue

**Comment by Miodec** [comment]
Yeah idk... I set my shield to the same settings and everything is still fine.. Can you try to purge all website data and try again?

**Comment by Miodec** [comment]
Tried importing the settings and everything is fine again...

**Comment by Caliv0** [comment]
I tinkered around a bit more.
I tried to reproduce this error on Windows 10, without success.
I switched to another machine which had Arch installed and the issue appeared again.

The issue appears very infrequent. About 1/3 of the time without my config, and 4/5 with my config.
Since disabling shield resolves the issue, I'll just do that.
